data-prepper-plugins/mongodb/src/main/java/org/opensearch/dataprepper/plugins/mongo/configuration/MongoDBSourceConfig.java

```diff
@@ -0,0 +1,109 @@
+package org.opensearch.dataprepper.plugins.mongo.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.validation.constraints.NotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class MongoDBSourceConfig {
+    private static final int DEFAULT_PORT = 27017;
+    private static final Boolean SSL_ENABLED = false;
```

Comment:
Why is the default false? We should aim for secure by default.

Reply:
Done